lib/Convos/Plugin.pm

```diff
@@ -0,0 +1,39 @@
+package Convos::Plugin;
+use Mojo::Base 'Mojolicious::Plugin';
+
+use Carp 'croak';
+
+sub register { croak 'Method "register" not implemented by subclass' }
```

Comment:
Yeah. I actually copy/pasted this line, when I still thought the plugins were supposed to be attached to Convos::Core.